src/data/padics/padic_norm.lean

```diff
@@ -95,21 +151,34 @@ begin
   rw [multiplicity.mul' hp', multiplicity.mul' hp']; simp
 end
 
+/--
+A rewrite lemma for `padic_val_rat p (q^k) with condition `q ≠ 0`.
+-/
 protected lemma pow {q : ℚ} (hq : q ≠ 0) {k : ℕ} :
     padic_val_rat p (q ^ k) = k * padic_val_rat p q :=
 by induction k; simp [*, padic_val_rat.mul _ hq (pow_ne_zero _ hq),
   _root_.pow_succ, add_mul]
 
+/--
+A rewrite lemma for `padic_val_rat p (q⁻¹)` with condition `q ≠ 0`.
+-/
 protected lemma inv {q : ℚ} (hq : q ≠ 0) :
   padic_val_rat p (q⁻¹) = -padic_val_rat p q :=
 by rw [eq_neg_iff_add_eq_zero, ← padic_val_rat.mul p (inv_ne_zero hq) hq,
     inv_mul_cancel hq, padic_val_rat.one]
 
+/--
+A rewrite lemma for `padic_val_rat p (q / r)` with conditions `q ≠ 0`, `r ≠ 0`.
+-/
 protected lemma div {q r : ℚ} (hq : q ≠ 0) (hr : r ≠ 0) :
   padic_val_rat p (q / r) = padic_val_rat p q - padic_val_rat p r :=
 by rw [div_eq_mul_inv, padic_val_rat.mul p hq (inv_ne_zero hr),
     padic_val_rat.inv p hr, sub_eq_add_neg]
 
+/--
+Gives a condition for `padic_val_rat p (n₁ / d₁) ≤ padic_val_rat p (n₂ / d₂),
```

Comment:
```suggestion
A condition for `padic_val_rat p (n₁ / d₁) ≤ padic_val_rat p (n₂ / d₂),
```
This is just a suggestion. The word "gives" refers to the lemma under consideration, but it still feels like it's dangling in the air a bit. Interested to hear what others think. (This suggestion would also apply to some other docstrings below.)